.NET/Microsoft.Recognizers.Definitions.Common/BaseCreditCard.cs

```diff
@@ -0,0 +1,29 @@
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+//
+//     Generation parameters:
+//     - DataFilename: Patterns\Base-CreditCard.yaml
+//     - Language: NULL
+//     - ClassName: BaseCreditCard
+// </auto-generated>
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// ------------------------------------------------------------------------------
+
+namespace Microsoft.Recognizers.Definitions
+{
+    using System;
+    using System.Collections.Generic;
+
+    public static class BaseCreditCard
+    {
+      public const string WordBoundariesRegex = @"\b";
+      public const string NonWordBoundariesRegex = @"\B";
+      public const string EndWordBoundariesRegex = @"\b";
+      public static readonly Func<string, string, string> CNCreditCardRegex = (WordBoundariesRegex, EndWordBoundariesRegex) => $@"((62|81)(\s?\d){{2}}(\s|-)?(\s?\d){{4}}(\s|-)?(\s?\d){{4}}(\s|-)?(\s?\d){{4}}(\s|-)?(\s?\d){{0,3}}|31(\s?\d){{2}}(\s|-)?(\s?\d){{4}}(\s|-)?(\s?\d){{4}}(\s|-)?(\s?\d){{4}}(\s|-)?(\s?\d){{3}})";
```

Comment:
Base should be used for all regexes, not only CN. 
Also, my understanding is that all numbers will fall in a similar range, and later prefixes can be checked to assign issuer, which also doesn't always directly map to country.
We should avoid any country mapping them. And card numbers can appear in any language.